sdk/src/index.ts

```diff
@@ -137,7 +137,7 @@ export interface MixinDeviceOptions<T> {
     });
   }
 
-  getMediaObjectConsole(mediaObject: MediaObject): Console {
+  getMediaObjectConsole(mediaObject: MediaObject): Console | undefined {
```

Comment:
I don't think this can ever return undefined. it will always return a console.

Reply:
Good point